source/docs/romi-robot/programming-romi-in-labview.rst

```diff
@@ -0,0 +1,37 @@
+Programming the Romi
+====================
+
+Writing a LabVIEW program for the Romi is very similar to writing a program for a regular FRC robot. In fact, all the same tools (LabVIEW, Driver Station, SmartDashboard, etc) can be used with the Romi.
+
+Creating a Romi Program
+-----------------------
+
+Creating a new program for a Romi is no different than creating a normal FRC program, similar to the :doc:`Zero To Robot </docs/zero-to-robot/step-4/index>` programming steps. Initially you may wish to create a separate project for use on just the Romi as the Romi hardware may be connected to different ports than on your roboRIO robot.
```

Comment:
I left the link to the index because that's what the other Romi article did but don't feel strongly. It makes sense to me that whatever we choose they should probably match?

Reply:
I think it's probably fine.